shell/platform/windows/cursor_handler.cc

```diff
@@ -45,6 +46,31 @@ void CursorHandler::HandleMethodCall(
     const auto& kind = std::get<std::string>(kind_iter->second);
     delegate_->UpdateFlutterCursor(kind);
     result->Success();
+  } else if (method.compare(kSetSystemCursorMethod) == 0) {
+    const auto& map = std::get<EncodableMap>(*method_call.arguments());
+    auto buffer = std::get<std::vector<uint8_t>>(
+        map.at(flutter::EncodableValue("buffer")));
+    auto width = std::get<int>(map.at(flutter::EncodableValue("width")));
+    auto height = std::get<int>(map.at(flutter::EncodableValue("height")));
+    auto hotx = std::get<double>(map.at(flutter::EncodableValue("hotx")));
+    auto hoty = std::get<double>(map.at(flutter::EncodableValue("hoty")));
```

Comment:
Please validate that the expected arguments are provided. Otherwise, return an error result

Reply:
Yes, I agree. Instead of getting the variable directly from std::get, we use `find` to check the variable existence.